### ISSUE TITLE
Compatibility with Photon UI

### ISSUE THREAD

**Issue by null-von-sushi** [horst3180/arc-firefox-theme#120]
Firefox is going to get a [new UI](http://www.omgubuntu.co.uk/2017/05/sneak-peek-firefox-photon-linux), and it would be nice to see Arc support it.

Currently one of the problems I noticed is that the page actions button does not display properly (top one is Arc Dark, bottom one is the default theme)

![theme](https://cloud.githubusercontent.com/assets/13616397/26233307/f4cbbef6-3c4b-11e7-9e59-f1790ca5e5cc.png)

**Comment by ArchangeGabriel** [comment]
Two things:

1. Photon will land when then won’t be working anymore (hence the `LEGACY` you’re seeing on those screenshots). So support for Photon makes no sense, at least in the current status of it.

2. The issue you’re seeing is not directly related to Photon (you don’t have Photon enabled on those screenshots), it’s a change that landed in Nightly where the Reload/Stop button was moved outside of the navbar and a new sharing button replaced it there (but this change is requested by Photon indeed). So both have new name/path, which are not provided by the Arc Theme. I’m not sure whether this could be fixed without breaking stable, I don’t know enough about theming. But they can probably be fixed in contrary to 1.

**Comment by ArchangeGabriel** [comment]
Also this is a duplicate of https://github.com/horst3180/arc-firefox-theme/issues/117.

**Comment by Noammac** [comment]
There seem to be new Photon problems.
With arc theme:
![image](https://user-images.githubusercontent.com/19552683/27260479-df7b8fe8-5435-11e7-8c48-8571161a419b.png)
Without arc theme:
![image](https://user-images.githubusercontent.com/19552683/27260457-741bba5c-5435-11e7-829a-db787e62afc5.png)

**Comment by ArchangeGabriel** [comment]
Yes, they have redone the whole hamburger menu. So this is expected (on a personal side, they also changed the downloads icon, and I absolutely do not like it w.r.t. the previous one)…

As I’ve said before, supporting the new UI is not really doable right now, since this UI will land in 57 when support for theming will at least change a lot, at worst disappear (hopefully temporarily).

**Comment by Noammac** [comment]
I see, so there's no way to fix the overflow menu right now.

**Comment by ArchangeGabriel** [comment]
I’ve got more news from the Mozilla stand at the RMLL. They said full theming won’t be supported at all starting with 57, only personas will remain.

Sad news for us here… :(

**Comment by jadenPete** [comment]
What the hell Mozilla?

**Comment by ArchangeGabriel** [comment]
They said that full theme are only used by a very tiny fractions of the users, so they are even less wanting to spend time supporting this that they do for extensions.

**Comment by lokesh-krishna** [comment]
Just wanted to say that the latest nightly looks gorgeous when just using the Arc theme as the GTK theme. I was so sad about losing theming with Firefox 57 but if this is how good it looks and how well it integrates with the GTK theme then I don't think this is going to be as much of a loss it might have been otherwise.
